inst/extdata/bic_animal_tissue_code.tsv

```diff
@@ -60,4 +60,6 @@ T66	Lung Powder	T66 Rat Lung Powder	t66-lung
 T67	Small Intestine Powder	T67 Rat Small Intestine Powder	t67-small-intestine
 T68	Liver Powder	T68 Rat Liver Powder	t68-liver
 T69	Brown Adipose Powder	T69 Rat Brown Adipose Powder	t69-brown-adipose
-T70	White Adipose Powder	T70 Rat White Adipose Powder	t70-white-adipose
\ No newline at end of file
+T70	White Adipose Powder	T70 Rat White Adipose Powder	t70-white-adipose
+T77	Lateral Gastrocnemius	T77 Rat Lateral Gastrocnemius	t77-lateral-gastroc

```

Comment:
Should it be "Lateral Gastrocnemius Powder" and "Vena Cava Powder" for consistency with all the other rat tissues,?

Reply:
nice catch again! Thanks!